src/server.js

```diff
@@ -29,7 +31,10 @@ module.exports = function() {
 	var app = express()
 		.use(allRequests)
 		.use(index)
-		.use(express.static("client"));
+		.use(express.static("client"))
+		.engine("html", expressHandlebars({extname: ".html", helpers: require("../client/js/libs/handlebars")}))
+		.set("view engine", "html")
+		.set("views", path.join(__dirname, "..", "client"));
```

Comment:
So, this is the way you are actually **meant** to do views in express. The way we were doing it manually caused us lots of weirdnesses, and when I tried to just throw our existing file into handlebars it didn't seem to work. so I decided to simplify it and use res.render. It works well and just a little bit of extra set up.